git/intermediate_git/a_deeper_look_at_git.md

```diff
@@ -34,67 +33,66 @@ Let's look at some ways we can change recent and distant history to fit our need
 
 Before we get started with the lesson, let's create a Git playground in which we can safely follow along with the code and perform history changing operations. Go to GitHub, and as you have in the past create a new repository. Call it whatever you'd like, and clone this repository to your local system. Now, let's `cd` into the repository we just cloned, and create some new files! Once you're in the repository follow along with the following commands. Look them up if you're confused about anything that's happening.
 
-~~~bash
+```bash
```

Comment:
Did you test this in our lesson preview tool? the markdown parser on our site uses ~~~ to delimit the start and end of code blocks

Reply:
This is probably changed by formatting plugins of vs code, unfortunately I don't know how to config them, I will close this pull request, thank you for your time.